### ISSUE TITLE
TypeError: type str doesn't define __round__ method

### ISSUE THREAD

**Issue by BertilJ** [filcole/home-assistant#13]
**Home Assistant release with the issue: 0.86 0**

**Last working Home Assistant release (if known): N/A tests ongoing**

**Operating environment (Hass.io/Docker/Windows/etc.): Hass.io**

**Component/platform: Nissan Leaf component**

**Description of problem:**

Got the component to work again, but now there seems to be new issues. Looks like the battery level isn't numerical.

**Problem-relevant `configuration.yaml` entries and (fill out even if it seems unimportant):**
```
nissan_leaf:
  username: "xxx"
  password: "xxx"
  nissan_connect: true
  region: 'NE'
  update_interval: 15
  update_interval_charging: 5
  update_interval_climate: 5
  force_miles: false
```

**Traceback (if applicable):**
```
File "/usr/local/lib/python3.6/site-packages/homeassistant/helpers/entity.py", line 232, in async_update_ha_state
    state = self.state
  File "/config/custom_components/sensor/nissan_leaf.py", line 51, in state
    return round(self.car.data[DATA_BATTERY], 0)
TypeError: type str doesn't define __round__ method
```

**Additional information:**

Carwings is most probably retrieving data for the battery since I turned on the SMS service from Nissan and I'm getting a SMS with the battery information from the site at each restart of Hass.

Also seems like the climate control switch is on even if it isn't (this can of course be due to Nissans data).

Device tracker, sensor.xxx_range and sensor.xxx_range_ac updates correctly, I can't yet say how the charging status and plug status works, need to see that once the car is connected again.

**Comment by filcole** [comment]
Thanks.  Think this should be fixed by release 2.7 of pycarwings.  Not able to test at the moment because Nissan Servers are down again :(

**Comment by BertilJ** [comment]
Great job!!

I managed to get this tested, seems that the issue is now fixed in 2.7.

If someone just could fix the Nissan servers we would be so much happier!!

Closing.
